src/tester.rs

```diff
@@ -524,6 +528,15 @@ fn run_tests(
                                     failure.status = Some("Error".to_owned());
                                 }
                             }
+                            Status::Signal => {
+                                if cfg!(unix) {
+                                    failure.status = Some("Did not exit due to signal".to_owned());
+                                } else {
```

Comment:
I think that in this case we can probably sensibly treat `Status::Signal` the same as `Status::Error`?

Reply:
I can see compelling arguments for both sides really. Ultimately, something that terminates with a signal in UNIX will probably also fail in windows with whatever their termination mechanism is. For the sake of compatibility with other platforms I think I agree with you.

Similarly, if we go down this route, we should probably aim for the same fallback-to-error behaviour with `Status::Int`.

Reply:
Maybe we should make `Status::Signal` an entirely compile-time switch so that on non-Unix platforms it's not even compiled in?

Reply:
What would the behaviour be if I ran a test with `status: signal` on a windows machine? A test ignored message in the runner output?

Reply:
I think in the parser we'd just convert `status: signal` to `Status::Int`.

Reply:
You mean to `Status::Error`?

Reply:
Mea culpa! Yes you're right.

Reply:
Hmm, I don't think that's really much different to the previous option. I think if we go to the effort of conditionally compiling `Status::Signal` related code, we should only run tests that rely on it on unix platforms.

Reply:
OK, well we do have support for ignoring tests, so I guess we could do that on non-Unix. I'm easy either way.

Reply:
I prefer that way. I can imagine a scenario where a program which terminates with a signal on Unix is valid on Windows. In which case, falling back to Status::Error would not be the correct behaviour. I'll push a commit that changes this.